src/controls/controlRendering.ts

```diff
@@ -124,9 +124,8 @@ export function renderSquareControl(
   // this is still wrong
   ctx.lineWidth = 1;
   ctx.translate(left, top);
-  //  angle is relative to canvas plane
-  const angle = fabricObject.getTotalAngle();
-  ctx.rotate(degreesToRadians(angle));
+  //  rotation is relative to canvas plane
+  ctx.rotate(calcPlaneRotation(fabricObject.calcTransformMatrix()));
```

Comment:
> So please keep using it where it was, since you removed the overhead of qrDecompose from it. The conversion from radians to degree is a multiplication and is not worth loosing the optimization for the common use case.

are you talking about this change??
I don't get your reason, what do you mean?
Why change to an angle and back to rad? It is purly redudant